src/cpprealm/internal/bridge/timestamp.hpp

```diff
@@ -31,16 +32,10 @@ namespace realm::internal::bridge {
         }
     private:
         static constexpr int32_t nanoseconds_per_second = 1000000000;
-#ifdef __i386__
-        std::aligned_storage<16, 4>::type m_timestamp[1];
-#elif __x86_64__
-        std::aligned_storage<16, 8>::type m_timestamp[1];
-#elif __arm__
-        std::aligned_storage<16, 8>::type m_timestamp[1];
-#elif __aarch64__
-        std::aligned_storage<16, 8>::type m_timestamp[1];
-#elif _WIN32
-        std::aligned_storage<16, 8>::type m_timestamp[1];
+#ifdef CPPREALM_HAVE_GENERATED_BRIDGE_TYPES
+        storage::Timestamp m_timestamp[1];
+#else
+        std::pair<int64_t, int32_t> m_timestamp;
```

Comment:
`Timestamp` is just a `int64_t` and a `int32_t` and there's a trivial constructor for conversion so we can simplify the storage.